### PR TITLE
Engelko hw12

### PR DESCRIPTION
**Pull request by Engelko**
Tasks:
1. Bash. Replace  "s3:DeleteBucket" with "s3:CreateBucket" in the fileA
2. Bash. Add new line "arn:aws:s3:::hw12-bucket"
 after  "arn:aws:s3:::backup-test*" in fileA
3. Create a bash script to compare 2 numbers(A and B). Use read to get variables to your script, like most of you did before :) . Output should be like A is greater than B OR A and B are equal OR A is less than B.
4. Create bash script as in point 3 but compare 3 numbers - A, B and C. Output should be the same as in the previous point, but only comparing 3 numbers with each other in pairs. 

For point 3 and 4 script should request input data until you type exit(e) instead of any number.

### CODE REVIEW

File: Bash_scripts/HW12/bash_script_3.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	echo "Please enter a number A "
+read NUM_A
+
+	echo "Please enter a number B "
+read NUM_B
+
+	echo "Please enter a number C "
+read NUM_C
+
+if (( $NUM_A > $NUM_B )) && (( $NUM_A > $NUM_C)); then
```
Comment (mavrik90) [inline Bash_scripts/HW12/bash_script_3.sh]: What about comparing B and C?

File: Bash_scripts/HW12/bash_script_3.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	echo "Please enter a number C "
+read NUM_C
+
+if (( $NUM_A > $NUM_B )) && (( $NUM_A > $NUM_C)); then 
+	echo "A is greater than B and A is a greater than C";
+
+elif (( $NUM_B > $NUM_A)) && (($NUM_B > $NUM_C)); then 
+	echo "B is greater than A and B is a greater than C";
+
+elif (( $NUM_A == $NUM_B)) && (($NUM_A == $NUM_C)); then
```
Comment (mavrik90) [inline Bash_scripts/HW12/bash_script_3.sh]: What if A = B not not C?

File: Bash_scripts/HW12/bash_script_3.sh (hunk 1 above)
Comment (Engelko) [reply Bash_scripts/HW12/bash_script_3.sh]: > What about comparing B and C?

in line 14, elif compares B and C

File: Bash_scripts/HW12/bash_script_3.sh (hunk 2 above)
Comment (Engelko) [reply Bash_scripts/HW12/bash_script_3.sh]: > What if A = B not not C?

Добавил возможность сравнивать равные числа. теперь все должно быть гуд.

Comment (mavrik90) [reply Bash_scripts/HW12/bash_script_3.sh]: please compare numbers with each other. A and B, B and C, A and C
For example A=1  B=10 C=6
```
A less than B, B greater than C, A less than C

File: Bash_scripts/HW12/bash_script_1.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+FILE=$1
+sed -i 's/s3:DeleteBucket/s3:CreateBucket/' $FILE
```
Comment (mavrik90) [inline Bash_scripts/HW12/bash_script_1.sh:3]: Where is line replacement?

**Comment by mavrik90** [comment]
Please provide screenshot proof for json file change

File: Bash_scripts/HW12/bash_script_1.sh (hunk 3 above)
Comment (Engelko) [reply Bash_scripts/HW12/bash_script_1.sh:3]: [https://i.imgur.com/pe2mbn7.png](url)

**Comment by Engelko** [comment]
> Please provide screenshot proof for json file change



> Please provide screenshot proof for json file change

[https://github.com/Engelko/DevOps_course/blob/Engelko-HW12/Bash_scripts/HW12/putty_j2cRstPqy1.png?raw=true](url)

**Comment by Engelko** [comment]
> Please provide screenshot proof for json file change
[Bash_scripts/HW12/putty_j2cRstPqy1.png](url)

**Comment by Engelko** [comment]
> Please provide screenshot proof for json file change

Иллюстрация к проекту

**Review by mavrik90 — APPROVED** [review]
LGTM

**Pull request merged**